yoyodyne/util.py

```diff
@@ -16,6 +16,18 @@ def log_info(msg: str) -> None:
     print(msg, file=sys.stderr)
 
 
+def log_warning(msg: str) -> None:
+    """Logs a warning msg to sys.stderr.
+
+    We can additionally consider logging to a file, or getting a handle to
+    the PL logger.
+
+    Args:
+        msg (str): the message to log.
+    """
+    log_info(Warning(msg))
```

Comment:
I am wondering if this is really useful to convert to a `Warning` object. I have not found them useful in general.

Reply:
Do you suggest just doing 

```
util.log_info(Warning(msg))
```
in the collator then?

Reply:
Oh, no, I am suggesting that this should not raise a warning at all, it should just log it to stderr like other things.